### PR TITLE
Create fr-FR.md

### PR DESCRIPTION
**Pull request by craftish37**
FRENCH TIME HERE

*It's not like you had 29 500 more downloads after all*

Anyway, here's a decent french translation, I'll recheck for any errors I could make but I'm pretty confident 🙂

### CODE REVIEW

File: Description/fr-FR.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Français (FR)
+
+Un modpack basé sur Fabric pour Minecraft orienté sur l'optimisation graphique et des performances. 
+
+[![Current review](https://img.youtube.com/vi/bb8G9X5Q_4I/maxresdefault.jpg)](https://www.youtube.com/watch?v=bb8G9X5Q_4I)
```
Comment (Madis0) [inline Description/fr-FR.md:5]: You forgot the alt text. If you can find a French review, you can replace the link as well.

**Comment by Madis0** [comment]
I'm gonna mark it as draft until you consider both files ready. Also this will be reviewed by our own French speaker before merge 🙂

**Comment by Julienraptor01** [comment]
i'll review when you think it's entirely fine

File: Description/fr-FR.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+> **Q**: Quelque chose ne va pas ! Vous devriez ajouter quelque chose !
+> 
+> **R**: [Postez-le sur notre GitHub.][8]
+
+
+> **Q**: Quel mods s'y trouvent ? 
+> 
+> **R**: Cliquez sur n'importe quelle version et descendez [ou regardez la table de comparaison][12].
+
```
Comment (Madis0) [inline Description/fr-FR.md:54]: There is a new entry here in upstream btw

**Comment by Madis0** [comment]
Thanks for the start of the translation, this will be used and continued by Julien to get French fully translated :tada:

**Pull request closed without merging**